### DISCUSSION TITLE
7

### DISCUSSION THREAD

**Discussion by giscus** [car-ffeine/car-ffeine.github.io#6] [Announcements]
# 7

프로젝트 브랜치명 컨벤션이 feat/이슈번호여서, 브랜치명에서 이슈번호만 가져온 다음 커밋할 때마다 커밋 메시지 아래단(footer)에 이슈 번호를 자동으로 입력해주고 싶었다. 자동으로 입력된다면 깜빡하고 이슈 번호를 안 적는 일도 없고, 시간도 단축할 수 있기 때문이다.

https://car-ffeine.github.io/7

**Comment by drunkenhw** [comment]
덕분에 아주 편하게 작업할 수 있겠군요!

**Comment by sosow0212** [comment]
우리팀 보안 전문가네요 👍
